src/pages/Article.jsx

```diff
@@ -0,0 +1,23 @@
+import { useLocation } from "react-router-dom";
+import ArticleList from "../components/article/ArticleList";
+import Header from "../components/Header";
+import Nav from "../components/Nav";
+import User from "../components/user/User";
+
+function Article() {
+  const location = useLocation();
+  const query = new URLSearchParams(location.search);
```

Comment:
React has it's own version of this called useSearchParams. The React docs for it aren't the best, but this article might help you: https://www.kindacode.com/article/react-router-useparams-usesearchparams-hooks/

I think if you're going to use searchParams/queries in React it would be better to use the React hook for it.